cypress/integration/1228-public_page/1231-owner_navigates_to_the_public_page_for_publication_for_published_publication_without_doi.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Owner navigates to the Public Page for Publication for published publication without DOI
+
+    @1231
+    Scenario: Owner navigates to the Public Page for Publication for published publication without DOI
+        Given that the Creator navigates to the Public Page for Publication
+        And they are the Owner of this Publication
+        And the Publication has no DOI
+        When they see the Status Bar
+        Then they see buttons for "Request a DOI" and "Edit Publication"
```

Comment:
trodde vi erstattet "Public Page for published publication" med "Landing page"? hva gjør vi med gamle scenario som ikke er oppdatert? skal vi bry oss om å endre de?

Reply:
Hmm, det er en jobb ihvertfall. Ideelt sett så burde vi det.